sound/soc/codecs/es8316.c

```diff
@@ -777,7 +772,7 @@ static void es8316_enable_jack_detect(struct snd_soc_component *component,
 		enable_irq(es8316->irq);
 		es8316_irq(es8316->irq, es8316);
 
-		initial = 1;
+		initial = 0;
```

Comment:
thanks for the patch @yangxiaohua2009 

isn't that change logically the same thing as before? !0 and 1 are not different...

It's usually a good idea to avoid static variables and use a context instead.